src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/MergeBuildManifestsTests.cs

```diff
@@ -0,0 +1,307 @@
+using System;
```

Comment:
Do we need the license statement on these guys?

Reply:
No idea, but I'll make sure all the new files have them.